libs/datamodel/core/src/transform/ast_to_dml/lift.rs

```diff
@@ -255,19 +264,6 @@ impl<'a> LiftAstToDml<'a> {
 
             attrs.append(&mut custom_type.directives.clone());
             Ok((field_type, attrs))
-        } else if USE_CONNECTORS_FOR_CUSTOM_TYPES {
```

Comment:
This branch was just a prototype for the other syntax that was suggested as an alternative.